translate.py

```diff
@@ -0,0 +1,12 @@
+
+
+class JokeTranslator:
+
+    #mock translator
+    @staticmethod
+    def get_joke(jokes, langInput):
```

Comment:
I'd probable make this more generic for flexibility. Something like:

```
class Translator:

    @staticmethod
    def translate(text: str, language: str) -> str:
          return "translated text"
```

Then later you can do something like:

```
translated_jokes = [Translator.translate(joke) for joke in jokes]
```

Reply:
I have done it a little bit different, let me know if that's ok.